contrib/nitiwiki/src/wiki_links.nit

```diff
@@ -199,6 +199,8 @@ redef class WikiArticle
 	# Checks if `self.name == "index"`.
 	fun is_index: Bool do return name == "index"
 
+	redef fun dir_href do return href
```

Comment:
This one might reopen #1872 in fact. the `href` is the path from root to the page, the dir_href is thepath from root to the directory of the page.